java/src/jmri/ConfiguringSystemConnectionMemo.java

```diff
@@ -0,0 +1,10 @@
+package jmri;
```

Comment:
Since the method this defines is only used inside the `jmri.jmrix` package, is there a reason not to define it there?

Reply:
It possibly could be defined there.  I’ll make that change.